site/snippets/email-css.php

```diff
@@ -714,8 +714,13 @@
 body.outlook p {
   display: inline !important;
 }
-<?php if($page->css() == 'breakground'): ?>
-  <?php snippet('email-css-breakground') ?>
+
+<?php if($page->css() == 'rallyday-au'): ?>
+  <?php snippet('email-css-rallyday-au') ?>
+<?php endif ?>
+
+<?php if($page->css() == 'rallyday-us'): ?>
```

Comment:
Lets change this to an else/if statement so we can expand it later. https://getkirby.com/docs/cookbook/php-templates#php-templates-101